### PR TITLE
Implement `scss_files` option

### PR DESCRIPTION
**Pull request by doits**
refs #4

### CODE REVIEW

**Comment by mmozuras** [comment]
@doits looks good :smile: 

Will you write some specs for it? At least for the `scss_file?` method? :smile:

**Comment by doits** [comment]
@mmozuras there you go

**Comment by doits** [comment]
Not sure why Travis' checks fail, locally they work. Do you have any idea?

**Comment by mmozuras** [comment]
@doits fetched it and it fails for me locally too.

**Comment by mmozuras** [comment]
Here's a fix that made it pass for me: https://github.com/mmozuras/pronto-scss/commit/93c57a422cbc52dc3c31af3d2f38b77324d499eb

Just FYI on how I did (for reference etc.). I moved into `test.git` folder and:

1) renamed `git` to `.git`.
2) ran `git init`.
3) removed the unnecessary files and left only the changes in aforementioned fix.
4) renamed `.git` to `git` (because you can't have a repo inside a repo, that's the only way I found to write these tests).
5) committed the change.

Apply the fix to your PR and it should be good to merge :smile:

File: spec/fixtures/test.git/git/COMMIT_EDITMSG [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Add a second file with errors
```
Comment (mmozuras) [inline spec/fixtures/test.git/git/COMMIT_EDITMSG]: Remove this file :smile:

**Comment by doits** [comment]
@mmozuras The steps you proposed didn't fix it on travis, though. I just checkout out `test.git` from master and added the new `world.css.scss`, did a `git commit` and didn't touch anything else. Looks like this worked and didn't break travis for now.

**Comment by mmozuras** [comment]
@doits thanks! 

Released as `0.5.3` :smile: :bow:

**Pull request merged**